app/views/newMain.scala.html

```diff
@@ -0,0 +1,39 @@
+@import assets.AssetsResolver
+@import views.ViewHelpers._
+@import switchboard.Switches
+@(
+	title: String,
+	description: Option[String] = None,
+)(implicit assets: AssetsResolver, request: RequestHeader, switches: Switches)
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta name="google-site-verification" content="nFrhJ3suC2OpKRasEkZyH1KZKpSZc-ofno_uunJgfvg" />
+
+	@if(description.isDefined) {
+		<meta name="description" content="@description" />
+		<meta property="og:description" content="@description" />
+	}
+
+	<title>@title</title>
+	<meta property="og:title" content="@title"/>
+
+	<link rel="shortcut icon" type="image/png" href="@assets("images/favicons/32x32.ico")">
```

Comment:
do we need this stuff if it's going to go away once we transpose into React-land?

Reply:
good call, removing.